lib/propolis/src/hw/qemu/fwcfg.rs

```diff
@@ -1057,6 +1057,158 @@ mod test {
 pub mod formats {
     use super::Entry;
     use crate::hw::pci;
+    use zerocopy::{Immutable, IntoBytes};
```

Comment:
this is like a hundred lines away from being large enough I'd kick it over to a `formats.rs`, but not quiiiite there yet..